database/src/main/java/com/firebase/ui/database/Timestamp.java

```diff
@@ -0,0 +1,25 @@
+package com.firebase.ui.database;
+
+import com.google.firebase.database.Exclude;
+import com.google.firebase.database.PropertyName;
+import com.google.firebase.database.ServerValue;
+
+import java.util.Map;
+
+public class Timestamp {
+    private long mTimestamp;
+
+    @PropertyName("timestamp")
+    private Map<String, String> getServerValue() {
+        return ServerValue.TIMESTAMP;
+    }
+
+    @Exclude
+    public long getTimestamp() {
+        return mTimestamp;
+    }
```

Comment:
I don't think we'd want to return system time, that's not the same as unset.  I prefer how it is now, it's the least opinionated and most likely to solve the original problem (serialization/deserialization of time stamps is annoying)